src/replxx_impl.cxx

```diff
@@ -807,12 +807,7 @@ void Replxx::ReplxxImpl::render( HINT_ACTION hintAction_ ) {
 	_utf8Buffer.assign( _data );
 	if ( !! _highlighterCallback ) {
 		IOModeGuard ioModeGuard( _terminal );
-		_highlighterCallback( _utf8Buffer.get(), colors );
-	}
-	paren_info_t pi( matching_paren() );
```

Comment:
It will be good to preserve old behavior of `_highlighterCallback` is not set, so that later we can backport the changes, though for this we need not to change `highlighter_callback_t`, but instead, introduce different callback to make it compatible, anyway let's fix at least this minor thing.